src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx

```diff
@@ -110,60 +99,69 @@ const ReleaseFootnotesSection = ({
   return (
     <>
       <h2>Footnotes</h2>
-      {!hasSufficientData && (
-        <p>
-          Before footnotes can be created, relevant data files need to be
-          uploaded. That can be done in the{' '}
-          <Link to="#data-upload">Data uploads section</Link>.
-        </p>
-      )}
-      {loading && <LoadingSpinner />}
-      {!loading && hasSufficientData && footnoteMeta && footnoteMetaGetters && (
-        <>
-          {canUpdateRelease && (
-            <FootnoteForm
-              {...footnoteForm}
-              footnote={undefined}
-              onOpen={footnoteFormControls.create}
-              onCancel={footnoteFormControls.cancel}
-              onSubmit={footnoteFormControls.save}
-              isFirst={footnotes && footnotes.length === 0}
-              footnoteMeta={footnoteMeta}
-              footnoteMetaGetters={footnoteMetaGetters}
-            />
-          )}
-          {footnoteMeta && (
-            <>
-              <FootnotesList
-                footnotes={footnotes}
-                footnoteMeta={footnoteMeta}
-                footnoteMetaGetters={footnoteMetaGetters}
-                footnoteFormControls={footnoteFormControls}
-                canUpdateRelease={canUpdateRelease}
+      {footnotesData === undefined ||
+        (!footnotesData.hasSufficientData && (
+          <p>
+            Before footnotes can be created, relevant data files need to be
+            uploaded. That can be done in the{' '}
+            <Link to="#data-upload">Data uploads section</Link>.
+          </p>
+        ))}
+      {footnotesData && loading && <LoadingSpinner />}
+      {footnotesData &&
+        !loading &&
+        footnotesData &&
+        footnotesData.hasSufficientData &&
+        footnotesData.footnoteMeta &&
```

Comment:
This `&&` chain is nasty. Put the `footnoteMeta` and `footnoteGetters` closer to where they're actually being used